templates/downloads/index.html

```diff
@@ -37,10 +37,61 @@ <h1 class="call-to-action">Download the latest version of Python</h1>
 {% endblock header_content %}
 
 {% block content %}
-                <div class="row download-list-widget">
-
+                <div class="row active-release-list-widget">
                     {% render_active_banner %}
 
+                    <h2 class="widget-title">Active Python Releases</h2>
+                    <p class="success-quote"><a href="https://devguide.python.org/#status-of-python-branches">For more information please visit the Python Developer's Guide</a>.</p>
+
+                    <div class="list-row-headings">
+                        <span class="release-version">Python Version</span>
+                        <span class="release-status">Maintenance Status</span>
+                        <span class="release-start">First Released</span>
+                        <span class="release-end">End of Support</span>
+                        <span class="release-pep">Release Schedule</span>
```

Comment:
Use sentence case for consistency with other table style:

```suggestion
                        <span class="release-version">Python version</span>
                        <span class="release-status">Maintenance status</span>
                        <span class="release-start">First released</span>
                        <span class="release-end">End of support</span>
                        <span class="release-pep">Release schedule</span>
```